### ISSUE TITLE
Allowing the "style" attribute

### ISSUE THREAD

**Issue by tekin** [tiff/wysihtml5#187]
Looking through how the library works, it doesn't look like there is a clear way to allow the style attribute on tags. Is this the case? I Just want to check before I dive in and try and patch things.

**Comment by DavidBradbury** [comment]
Inline styles = Bad news bears

**Comment by tekin** [comment]
@DavidBradbury not if you're using the editor for HTML emails! ;)

**Comment by DavidBradbury** [comment]
I'd argue that HTML e-mails are broken entities in themselves! That said, I will admit there is an inherent usefulness in being able to allow inline styles even if it makes me grumble about style/content separation and semantics. :)

**Comment by Bratzilla** [comment]
I agree with the OP. There's what is in theory "perfect" practice, and then there's the "real world." Inline styles may be bad news, but unfortunately there are cases where using the class approach may not be feasible. In addition to HTML email, there is also the issue with people who may want to use this editor who already have a database full of articles, etc., with inline styles. Unless there is another approach (and I'd love to hear it), I cannot fathom someone slogging through hundreds of not thousands of entries trying to fix all that.

At the very least, inline styles should be an option.

**Comment by edslocomb** [comment]
We're in the situation Bratzilla describes; we've got a lot of documents in our database that use &lt;style&gt; tags and define class names ad-hoc therein.  And some that use style="" attributes.  And some that use both.

None of these will survive a trip through wysihtml5's parser.

**Comment by ohhaiohhai** [comment]
Take a look at my comments on this other similiar thread:
https://github.com/xing/wysihtml5/issues/212

**Comment by celmaun** [comment]
+1 will come in handy.

**Comment by russomi** [comment]
+1 -- added this commit to my local version to support inline styles.

**Comment by jaybee** [comment]
Glad someone found this useful :)

**Comment by badzeugene** [comment]
i done this, but still not working. is this also apply to insertHtml?
